brands/amenity/bank.json

```diff
@@ -4530,6 +4575,28 @@
       "name": "Security Bank"
     }
   },
+  "amenity/bank|Security Service Federal Credit Union": {
+    "countryCodes": ["us"],
+    "matchNames": ["security service fcu"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "Security Service Federal Credit Union",
+      "brand:wikidata": "Q7444993",
+      "brand:wikipedia": "en:Security Service Federal Credit Union",
+      "name": "Security Service Federal Credit Union",
+      "short_name": "SSFCU"
+    }
+  },
+  "amenity/bank|Service Credit Union": {
+    "countryCodes": ["de", "us"],
```

Comment:
Are you sure this is in both europe and the US?

Reply:
Surprising but per [Wikipedia](https://en.wikipedia.org/wiki/Service_Credit_Union) (emphasis mine):
>As of 2018, Service Credit Union has 50 locations (35 in New Hampshire, 1 in Massachusetts, 1 in North Dakota **and 13 on US military installations in Germany**)

Reply:
That actually makes perfect sense. I'm sure USAA probably has a few branches in Germany for the same reason.